modules/kubernetes-addons/gatekeeper/locals.tf

```
@@ -0,0 +1,29 @@
+locals {
+  name      = "gatekeeper"
+  namespace = "gatekeeper-system"
+
+  default_helm_config = {
+    name             = local.name
+    chart            = local.name
+    repository       = "https://open-policy-agent.github.io/gatekeeper/charts"
+    version          = "3.9.0"
+    namespace        = local.namespace
```

since its only used once, we don't need to set a local for it (and remove line 3)
```suggestion
    namespace        = "gatekeeper-system"
```

---

Fixed